### PR TITLE
Added SMBIOS version in BIOS info

### PR DESCRIPTION
**Pull request by chatenilesh**
Added support for Redhat release in OS info.
Added support for SMBIOS version in BIOS info.

### CODE REVIEW

**Comment by zcalusic** [comment]
Hello @chatenilesh and thanks for your contribution!

Unfortunately, lots of changes would be required, before I can accept it.

First of all, two completely different parts of code have been updated with a single patch/PR. Usually, this is not a good idea, because it makes it hard to follow git history later, possibly revert bad changes etc...

Apropos the first part, better support for Red Hat, yeah, it was never written because I don't have access to any Red Hat machine. Still, I'd expect the support for RHEL eventually to be along the lines of CentOS, 'cause the two should be very similar. To follow what has already been written, I'd like os-release file generated if it doesn't exist yet (so it's there for next sysinfo pass, and other software to use...), and I'd expect more accurate regexp, so we can catch corner cases.

Actually, I'd be willing to help with this part, if you have access to multiple versions od Redhat, and can send pairs of os-release and redhat-release files, I'd be willing to add support myself, if that's OK with you?

Apropos the second part, detecting SMBIOS version, I find that one even more problematic:
- you've pulled the first dependency in, just to get SMBIOS version? If we could live this long without a single dependency, I'd like it to stay this way for as long as possible. It's kind of neat. :)
- all labels in JSON have been lowercase so far, and suddenly SMBIOS version is CamelCase?
- count me curious, what are real life use cases for SMBIOS version?

To comment on the last point, even if there probably will be new info added in the future, I'd still like to keep to really useful set.

**Comment by chatenilesh** [comment]
Makes sense, I can make the recommended changes.
I liked the way this library was written and it stands out wrt other libraries out there but while browsing the code I found out that you are using dmidecode to get memory info. I think parsing dmidecode is in itself is a mini project and I think instead of rewriting the whole dmidecode parsing in this library we could use the https://github.com/digitalocean/go-smbios. I we switch to this, adding attributes like memory serial number etc would be easy.

I wanted some other system info during which I required smbios version so I added it in the bios section of this library.

I have tested this library on few RHEL version and it looks fine, would need minor changes of which I added one. I would be more than happy to help you to added RHEL support.

**Comment by chatenilesh** [comment]
@zcalusic Hi, could you check the new pull request for RHEL OS version. https://github.com/zcalusic/sysinfo/pull/10
